packages/mobile/src/screens/chat-screen/ChatListItem.tsx

```diff
@@ -105,34 +52,64 @@ export const ChatListItem = ({ chatId }: { chatId: string }) => {
   return (
     <TouchableHighlight onPress={handlePress}>
       {otherUser ? (
-        <View style={styles.root}>
-          <View style={styles.contentRoot}>
-            <View style={styles.userContainer}>
+        <Flex
+          column
+          pv='l'
+          ph='xl'
+          backgroundColor='white'
+          borderBottom='default'
+          w='100%'
+        >
+          <Flex row justifyContent='space-between' w='100%' gap='s'>
+            <Flex row flex={1} justifyContent='space-between'>
               <ProfilePicture
                 userId={otherUser.user_id}
                 style={css({ width: spacing.unit12, height: spacing.unit12 })}
               />
-              <View style={styles.userTextContainer}>
-                <View style={styles.userNameContainer}>
-                  <Text style={styles.userName}>{otherUser.name}</Text>
+              <Flex column pt='2xs' ml='s' mb='s' flex={1}>
+                <Flex row mb='xs' wrap='nowrap'>
+                  <Text
+                    size='l'
+                    strength='strong'
+                    numberOfLines={1}
+                    flexShrink={1}
+                  >
+                    {otherUser.name}
+                  </Text>
                   <UserBadges user={otherUser} hideName />
-                </View>
-                <Text style={styles.handle}>@{otherUser.handle}</Text>
-              </View>
-            </View>
+                </Flex>
+                <Text size='s' numberOfLines={1}>
+                  @{otherUser.handle}
+                </Text>
+              </Flex>
+            </Flex>
             {chat?.unread_message_count && chat?.unread_message_count > 0 ? (
-              <View>
-                <View style={styles.unreadCountContainer}>
-                  <Text style={styles.unreadCount}>
+              <Box style={css({ flexShrink: 0 })}>
+                <Flex
+                  pv='xs'
+                  ph='s'
+                  borderRadius='xs'
+                  backgroundColor='accent'
+                  justifyContent='center'
+                  alignItems='center'
+                >
+                  <Text
+                    variant='heading'
+                    size='xs'
+                    textTransform='uppercase'
+                    strength='strong'
+                    color='staticWhite'
+                    style={css({ letterSpacing: 0.5 })}
+                  >
                     {clipMessageCount(chat?.unread_message_count ?? 0)}{' '}
```

Comment:
oof not your code (my code lol) but should probably include this whole thing including the whitespace into `messages.new`